orcid-api-web/src/main/java/org/orcid/api/memberV3/server/MemberV3ApiServiceImplHelper.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * =============================================================================
+ *
+ * ORCID (R) Open Source
+ * http://orcid.org
+ *
+ * Copyright (c) 2012-2014 ORCID, Inc.
+ * Licensed under an MIT-Style License (MIT)
+ * http://orcid.org/open-source-license
+ *
+ * This copyright and license information (including a link to the full license)
+ * shall be included in its entirety in all copies or substantial portion of
+ * the software.
+ *
+ * =============================================================================
+ */
+package org.orcid.api.memberV3.server;
+
+import org.orcid.core.exception.PutCodeFormatException;
+
+/**
+ * 
+ * @author Shobhit Tyagi
+ * 
+ */
+public class MemberV3ApiServiceImplHelper {
```

Comment:
https://trello.com/c/nfrgXkeU/3973-remove-memberv2apiserviceimplhelper-and-memberv3apiserviceimplhelper easy win @bobcaprice if you want to take it and do it for this branch, if not, no worry, we will eventually do it.